README.md

```diff
@@ -125,55 +125,41 @@ int foo(int a) {
 program-markers --mode=vr test.c --
 
 cat test.c | clang-format 
-#if defined DisableVRMarkerLE0_
-#define VRMARKERMACROLE0_(VAR)
-#elif defined UnreachableVRMarkerLE0_
-#define VRMARKERMACROLE0_(VAR)         \
-  if ((VAR) <= VRMarkerConstantLE0_)   \
+#if defined DisableVRMarker0_
+#define VRMARKERMACRO0_(VAR)
+#elif defined UnreachableVRMarker0_
+#define VRMARKERMACRO0_(VAR)                                                   \
+  if (!(VRMarkerLowerBound0_ <= (VAR) && (VAR) <= VRMarkerUpperBound0_))       \
     __builtin_unreachable();
 #else
-#define VRMARKERMACROLE0_(VAR)         \
-  if ((VAR) <= VRMarkerConstantLE0_)   \
-    VRMarkerLE0_();
-void VRMarkerLE0_(void);
-#endif
-#ifndef VRMarkerConstantLE0_
-#define VRMarkerConstantLE0_ 0
+#define VRMARKERMACRO0_(VAR)                                                   \
+  if (!(VRMarkerLowerBound0_ <= (VAR) && (VAR) <= VRMarkerUpperBound0_))       \
+    VRMarker0_();
```

Comment:
Do you intend to add macro that adds an else-branch marker as well?
This would allow to detect the case when the if-statement is completely dead in one compilation.

Reply:
The current idea is to rely on `.track_reachable_markers` to find the unreachable `VRMarkers` (for CSmith programs) as this is more reliable

Reply:
:+1:
I was specifically thinking about the range bisection but because this tests the interval `)a, b(` and not `[a,b]` it's still fairly easy to do.